### DISCUSSION TITLE
Directus automatically stops serving assets [from S3 Bucket] after certain amount of time(random).

### DISCUSSION THREAD

**Discussion by omkardedge011** [directus/directus#18007] [Archive]
We are using a S3 bucket to store the assets, but after certain amount of time the assets are not accessible and the request shows timeout.

Version:  "directus": "^9.23.4"

![image](https://user-images.githubusercontent.com/75299473/229064269-b54b7879-cc58-45e3-8b1c-852118d3c365.png)

However directus works correctly after restarting the process or server.

**Comment by rijkvanzanten** [comment]
Duplicate of https://github.com/directus/directus/issues/17983
